.github/workflows/dotnet.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: artifacts
-          path: ./artifacts/*.nupkg
+          path: ./artifacts/*.?(s)nupkg
```

Comment:
You have to split the push here.

Reply:
https://github.com/GDATASoftwareAG/motornet/blob/master/.github/workflows/dotnet.yml#L90 this also doesn´t work, see https://github.com/GDATASoftwareAG/motornet/runs/5388989422?check_suite_focus=true

Reply:
Are you sure? `*.?(s)nupkg` should match `*.nupkg` and `*.snupkg`. But I'm don't know if Github actions are implementing globbing correct.